doc/enterprise-shield/p_enterprise_shield_config.md

```diff
@@ -0,0 +1,418 @@
+      <ssl.verify-client>required</ssl.verify-client>
+      <ws.inactivity.timeout>60 seconds</ws.inactivity.timeout>
+    </connect-options>
+
+    <cross-site-constraint>
+      <allow-origin>*</allow-origin>
+    </cross-site-constraint>
+
+  </service>
+```
+
+As you can see, the `connect-options` are exactly the same for both the App1 and App2 services.
+
+You've completed configuring the DMZ Gateway! Next we will configure security for the connection between the DMZ and internal Gateways.
+
+Step 6: Configure Security Between the DMZ and Internal Gateways
+------------------------------------------------------------------------------------------
+
+Each Gateway maintains two databases for SSL connections: a keystore that stores SSL certificates to present to a client, and a truststore that stores keys from trusted sources. The following example shows the security block in the Gateway configuration file:
+
+``` xml:
+  <security>
+
+    <keystore>
+      <type>JCEKS</type>
+      <file>dmz.keystore</file>
+      <password-file>dmz.keystore.pw</password-file>
+    </keystore>
+
+    <truststore>
+      <type>JCEKS</type>
+      <file>dmz.keystore</file>
+      <password-file>dmz.truststore.pw</password-file>
+    </truststore>
+
+  </security>
+```
+
+**Note:** The one keystore file contains certificates for both the [frontplane](http://tech.kaazing.com/documentation/kaazing-glossary.html#frontplane-and-backplane) and [backplane](http://tech.kaazing.com/documentation/kaazing-glossary.html#frontplane-and-backplane) connections, which need a certificate to be presented. Similarly, the truststore contains keys for trusted sources for the frontplane and backplane connections. However, because Web-based clients typically are not configured to present client certificates, in practice, the truststore keys only apply to the backplane connections.
+
+
+These security concepts and configuration are described in detail in:
+
+-   [Require Clients to Provide Certificates to the Gateway](../security/p_tls_mutualauth.md)
+-   [Secure the Gateway Using Trusted Certificates](../security/p_tls_trusted.md)
+
+Step 7: Verify the Gateways are Running
+---------------------------------------------------------------------
+
+Start each Gateway and open a browser on your client to test your application.
+
+Step 8: Close Inbound Ports
+----------------------------------------------------
+
+Close the inbound ports on your firewall using the instructions provided by your system.
+
+Step 9: Verify the End-to-End Configuration
+-------------------------------------------------------------------
+
+Use a client within the DMZ to test out the internal Gateway before deploying Enterprise Shield™ in your production environment. You can repeat the instructions in [Step 7: Verify the Gateways are Running](#step-7-verify-the-gateways-are-running) to confirm everything is working properly.
```

Comment:
Originally I think we added this step to check that everything was still working after you closed the inbound ports.  But, that was a long time ago when ES was new to everyone and maybe Step 9 is not really necessary?  

We should probably chat about removing Step 9 or changing it so that we test from the clients instead. I need some more input from you on this one.  Thanks.